enterprise/backend/test/metabase_enterprise/audit_db_test.clj

```diff
@@ -32,7 +33,7 @@
     (testing "Audit DB content is not installed when it is not found"
       (t2/delete! :model/Database :is_audit true)
       (with-redefs [audit-db/analytics-dir-resource nil]
-        (is (= nil audit-db/analytics-dir-resource))
+        (is (= nil @#'audit-db/analytics-dir-resource))
```

Comment:
Nit and it's already this way in the existing code, but `(nil? ...)` is very slightly more concise.

Reply:
Agree. Going to merge and put up a separate PR for this so that I don't need to wait for CI to run again before starting the backport